Assets/GridTester.cs

```diff
@@ -6,23 +6,23 @@
 public class GridTester : MonoBehaviour
 {
 
-    private Grid grid;
+    [SerializeField] private HeatMapVisual heatMapVisual;
+
+    private BattleGrid grid;
+
     
    private void Start()
     {
-        grid = new Grid(4, 2, 10f, new Vector3(20,0));
+        grid = new BattleGrid(10, 10, 10f, Vector3.zero);
```

Comment:
magic numbers?

Reply:
All except one are not constants; these are values that we will be passing to instantiate a grid and it's size. Depending on the situation we might have larger or smaller grids. The exception is the size of the individual cell; we'll want to keep that consistent for art assets (importing tilesets, characters).

Reply:
As we haven't decided on the size of the sprites or tilesets and this is a test class I haven't defined it as a const yet.

Reply:
dont talk to me like that